resources/clients/react/elwing/src/setupTests.ts

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
```

Comment:
// For more information, see https://github.com/testing-library/jest-dom.

Reply:
See above.

Reply:
Got the 'OK' from Liz to leave this alone.